### PR TITLE
[Trusted Types] Fixing scriptloading trusted type violations

### PR DESCRIPTION
**Pull request by youssef-attia**
## What is the purpose of this pull request?

PR to fix violations specifically coming from scriptloading functions to help with making CKEditor [Trusted Types](https://developer.mozilla.org/en-US/docs/Web/API/Trusted_Types_API) compatible.

### CODE REVIEW

File: ckeditor.js
Comment (eozmen410) [inline ckeditor.js:1]: This looks good for fixing the TT violation, but should we use the legacy conversion function for this as well?

**Review by eozmen410 — CHANGES_REQUESTED** [review]

File: ckeditor.js
Comment (youssef-attia) [reply ckeditor.js:1]: Yep, you're right I'll copy over some of the tooling.

Comment (eozmen410) [reply ckeditor.js:1]: We can also merge the legacy conversion PR and then rebase this one, it might be easier to prevent any merge conflicts

Comment (youssef-attia) [reply ckeditor.js:1]: Replaced with htmlSafeByReview

**Review by eozmen410 — APPROVED** [review]

**Pull request merged**